zulip_bots/zulip_bots/terminal.py

```diff
@@ -3,7 +3,7 @@
 import sys
 import argparse
 
-from zulip_bots.run import import_module_from_source
+from zulip_bots.finder import import_module_from_source
```

Comment:
This should happend in the first commit where you move the function to `import_module_from_source`. The idea is to keep commits as atomical elements that work on their own. If, for example, we would now commit the first commit only, the tests would fail for zulip-terminal.